src/pages/wallet/SendPage.tsx

```diff
@@ -343,12 +348,7 @@ const SendPage = () => {
     }
   }, [chain, trigger, recipient])
 
-  const filteredAssets = useMemo(
-    () => availableAssets.filter(({ symbol }) => !symbol.endsWith("...")),
-    [availableAssets]
-  )
-
-  const assetsByDenom = filteredAssets.reduce(
+  const assetsByDenom = availableAssets.reduce(
```

Comment:
```suggestion
  const assetsByDenom = filteredAssets.reduce(
```

Reply:
@ericHgorski what was the reason for changing this line?